### PR TITLE
feat: add current markers visualization

### PR DESCRIPTION
**Pull request by polvalente**

### CODE REVIEW

File: boat_visualizer/lib/boat_visualizer_web/endpoint.ex [hunk 1]
```diff
@@ -10,41 +10,46 @@ defmodule BoatVisualizerWeb.Endpoint do
...
+  )
 
-  plug Plug.MethodOverride
-  plug Plug.Head
-  plug Plug.Session, @session_options
-  plug BoatVisualizerWeb.Router
+  plug(Plug.MethodOverride)
+  plug(Plug.Head)
+  plug(Plug.Session, @session_options)
+  plug(BoatVisualizerWeb.Router)
```
Comment (ChristianTovar) [inline boat_visualizer/lib/boat_visualizer_web/endpoint.ex]: Do you have something in your editor forcing macros to use parentheses?

Comment (polvalente) [reply boat_visualizer/lib/boat_visualizer_web/endpoint.ex]: Probably VSCode not handling nested projects correctly. I have a custom config for this in the Nx workspace but not for this project

Comment (polvalente) [reply boat_visualizer/lib/boat_visualizer_web/endpoint.ex]: I'll undo this

File: boat_visualizer/lib/boat_visualizer_web/live/map_live.ex [hunk 2]
```diff
@@ -14,29 +16,122 @@ defmodule BoatVisualizerWeb.MapLive do
...
-     |> assign(:current_min_position, 0)
-     |> assign(:current_max_position, Enum.count(coordinates) - 1)
-     |> assign(:show_track, true)}
+    {now, us} = NaiveDateTime.utc_now() |> NaiveDateTime.to_gregorian_seconds()
+    now = now + us / 1_000_000
+
+    min_lat = 42.1666
+    max_lat = 42.4093
+    min_lon = -71.0473
+    max_lon = -70.8557
```
Comment (ChristianTovar) [inline boat_visualizer/lib/boat_visualizer_web/live/map_live.ex:25]: Is this something we could calculate dynamically? Perhaps the way `Animation.set_map_view/1` could be used as reference for that. Either way, let's have it as something to tackle in the future once we add the ability to support other csv files besides the one we have hardcoded

**Review by ChristianTovar — APPROVED** [review]

File: boat_visualizer/lib/boat_visualizer_web/endpoint.ex (hunk 1 above)
Comment (ChristianTovar) [reply boat_visualizer/lib/boat_visualizer_web/endpoint.ex]: No biggie, thanks though!

File: boat_visualizer/lib/boat_visualizer_web/live/map_live.ex (hunk 2 above)
Comment (polvalente) [reply boat_visualizer/lib/boat_visualizer_web/live/map_live.ex:25]: We have the base code for this already, whenever the map changes I send an event updating the bounding box. We'd just have to associate it with map load as well

Comment (polvalente) [reply boat_visualizer/lib/boat_visualizer_web/live/map_live.ex:25]: Should be an easy update, but we should probably leave this for a follow up PR

**Review by schrockwell — APPROVED** [review]

**Pull request merged**